src/styles/common/StyledButton.ts

```diff
@@ -187,12 +189,25 @@ export const ButtonTheme = variants<'kind', keyof typeof ButtonVariations>('mode
 
       &:hover {
         color: ${white};
-        background-color: darkpurple;
+        background: darkpurple;
       }
     `,
   },
 })
 
+/*
+TODO: consider adding:
+  &.small {
+    font-size: 0.6rem;
+    padding: 0.3rem 0.5rem;
+  }
+
+  &.big {
+    font-size: 1.2rem;
+    padding: 0.65rem 1rem;
+  }
+*/
+
 export default styled.button`
   border: ${buttonBorder};
 
```

Comment:
Can this whole StyledButton.ts be kept dryer so that instead doing:

```
   warning: {
    light: css`
      color: ${white};
      background: ${warningLight};
      &:hover {
        background: ${warningDark};
        border-color: ${warningDark};
      }
    `,
    dark: css`
      color: ${white};
      background: ${warningDark};
      &:hover {
        background: ${warningLight};
        border-color: ${warningLight};
      }
    `,
  },
```

doing

```
  warning: {
    css`
      color: ${white};
      background: ${warningLight};
      &:hover {
        background: ${warningDark};
        border-color: ${warningDark};
      }
    `
  },
```

Then e.g. we could have a variable `warningDefault` which conditionally returns a color, based on getting the current mode (either dark or light).

This keeps this styling code dryer for defaults. For overrides we can specifically override the colors in a individual component. Let me know your thoughts.